config.rb.sample

```diff
@@ -42,6 +42,10 @@
 #   export DOCKER_HOST='tcp://127.0.0.1:2375'
 #$expose_docker_tcp=2375
 
+# Enable NFS sharing of your home directory to CoreOS
```

Comment:
Document what env var it looks for and where the homedir will be made available.